.github/workflows/nightly-conda.yml

```diff
@@ -1,15 +1,22 @@
 name: TorchArrow Nightly Build and Test for Conda
 on:
-#  schedule:
-#    - cron: "0 0 * * *"
-  pull_request:
-    branches:
-      - main
-      # For PR created by ghstack
-      - gh/*/*/base
+   schedule:
+      - cron: "0 0 * * *"
 
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+   # Use the following to trigger test on a dev PR
+   # pull_request:
+   #  branches:
+   #    - main
+   #    # For PR created by ghstack
+   #    - gh/*/*/base
+
+   # Allows you to run this workflow manually from the Actions tab
+   workflow_dispatch:
+
+   workflow_call:
+      secrets:
+         CONDA_NIGHTLY_PYTORCHBOT_TOKEN:
+            required: false
```

Comment:
You don't need `workflow_call` unless you want to invoke this workflow by another workflow.